core/test/integration/model/model_tags_spec.js

```diff
@@ -102,6 +102,18 @@ describe('Tag Model', function () {
                 })
                 .catch(done);
         });
+
+        it('can strip invisible unicode from slug', function (done) {
+            const tag = Object.assign(_.omit(testUtils.DataGenerator.forModel.tags[0], 'id'), {
+                slug: 'abc\u0008',
+            });
+            models.Tag.add(tag, context)
```

Comment:
This assertion could be returned, in that case, no need to pass `done` around. Something like the following:
```
            return models.Tag.add(tag, context)
                .then(function (newTag) {
                    should.exist(newTag);
                    newTag.get('slug').should.equal('abc');
                });
```